src/__tests__/expressionFunctions.js

```diff
@@ -53,7 +53,9 @@ test('ein', (t) => {
 test('email', (t) => {
   t.true(expFns.email().indexOf('@') !== -1);
   t.true(expFns.email().indexOf('.') !== -1);
-  t.true(expFns.email('intuit.com').indexOf('intuit.com') !== -1);
+  t.true(expFns.email('intuit.com').indexOf('@intuit.com') !== -1);
+  t.true(expFns.email('intuit.com', true).indexOf('_') !== -1);
```

Comment:
I guesss the downside of ava is that somebody would have no idea why you are testing against _ in the email string.
How can we make it more clear? Describe/it blocks or comments would help